docs/cloud-providers/aws/aws-provider.md

```diff
@@ -34,12 +34,101 @@ You can confirm the existense of the  [`aw provider`] by running:
 kubectl -n crossplane-system get provider/aws-provider
 ```
 
+## Optional: Setup AWS Provider Manually
```

Comment:
Up at line 4 in this doc it says `aw provider` when it should say `aws provider`. Would you mind fixing that in this PR?

Reply:
sure thing!